src/Terrain/PlanetWrenderer.h

```diff
@@ -1,46 +1,28 @@
 // "PlanetRenderer is a little too boring" -- Capital Asterisk, 2018
 #pragma once
 
-#include <cstdint>
-#include <iostream>
-#include <sstream>
-#include <string>
-
-#include <Urho3D/Container/Ptr.h>
-#include <Urho3D/Core/CoreEvents.h>
-#include <Urho3D/Engine/Engine.h>
-#include <Urho3D/Graphics/Camera.h>
-#include <Urho3D/Graphics/DebugRenderer.h>
 #include <Urho3D/Graphics/Geometry.h>
 #include <Urho3D/Graphics/Graphics.h>
 #include <Urho3D/Graphics/IndexBuffer.h>
-#include <Urho3D/Graphics/Light.h>
-#include <Urho3D/Graphics/Material.h>
 #include <Urho3D/Graphics/Model.h>
-#include <Urho3D/Graphics/Octree.h>
 #include <Urho3D/Graphics/Renderer.h>
-#include <Urho3D/Graphics/Skybox.h>
-#include <Urho3D/Graphics/StaticModel.h>
 #include <Urho3D/Graphics/VertexBuffer.h>
-#include <Urho3D/Input/InputEvents.h>
-#include <Urho3D/Input/Input.h>
-#include <Urho3D/IO/Log.h>
-#include <Urho3D/Math/MathDefs.h>
+
 #include <Urho3D/Resource/ResourceCache.h>
-#include <Urho3D/Resource/XMLFile.h>
-#include <Urho3D/Scene/SceneEvents.h>
-#include <Urho3D/Scene/Scene.h>
-#include <Urho3D/UI/Button.h>
-#include <Urho3D/UI/Font.h>
-#include <Urho3D/UI/Text.h>
-#include <Urho3D/UI/UIEvents.h>
-#include <Urho3D/UI/UI.h>
+
+#include <Urho3D/Container/Ptr.h>
+
+#include <Urho3D/IO/Log.h>
+
+#include <Urho3D/Core/CoreEvents.h>
+
+#include <cstdint>
 
 using namespace Urho3D;
 
 // The 20 faces of the icosahedron (Top, Left, Right)
 // Each number pointing to a vertex
-static constexpr const uint8_t sc_icoTemplateTris[20 * 3] {
```

Comment:
For variables, constexpr automatically means const.

This is not the case with member functions, however.